### DISCUSSION TITLE
CloudPingTest always pings one server, http.speedtest.las1.coreweave.com, on the CoreWeave page.

### DISCUSSION THREAD

**Discussion by lopatoid** [VarunAgw/CloudPingTest#45] [Q&A] [answered]
I think there is an error[ in this line](https://github.com/VarunAgw/CloudPingTest/blob/7dccf7f4f58334b0b7612681402b483c43f5cb67/coreweave.php#L12)

**Comment by leglock** [comment, marked as answer]
This is resolved with #53.
